### DISCUSSION TITLE
Rescheduling only some operations for product

### DISCUSSION THREAD

**Discussion by tjheiska** [frePPLe/frepple#471] [Q&A] [answered]
I have modeled the production having multiple stages by creating an operation producing an intermediate item for each stage. Then the next stage will use this item as operation material. This seems to work fine, if I schedule all the stages automatically by frePPLe. But, if I first schedule all stages, lock some stage by setting it "approved" or "confirmed", stretch it a bit and reschedule other stages, those stages overlap. 
I would expect the rescheduling fail with error if operation cannot be rescheduled without overlapping. Or do you think there's something wrong with my model or maybe this a limitation of Community Edition?

![image](https://user-images.githubusercontent.com/32795122/149289993-d317d470-9553-4dec-96f1-9fc381295eff.png)

**Comment by jdetaeye** [comment, marked as answer]
>>... or maybe this a limitation of Community Edition?

Correct.  The solver in the Community Edition doesn't have the functionality to "move existing manufacturing orders around".  It only "proposes new manufacturing orders".

**Reply by tjheiska** [reply]
Ok, so https://github.com/frePPLe/frepple/discussions/443 applies here also. In addion to "move existing manufacturing orders around" it won't take those as constraints when rescheduling other stages?

**Reply by jdetaeye** [reply]
The solver in the Community Edition will still plan around these. But there are situations where there just isn't a feasible solution to respects all material, capacity & lead time constraints without "moving around" the existing manufacturing orders.
